src/index.js

```diff
@@ -1,78 +1,226 @@
+import AppBar from './app-bar';
```

Comment:
They are bad for files that perform logic, true. but for files that the only thing they do is re-export i think it makes sense. without this syntax the number of lines become excessively large and hard to maintain.